src/themes/DefaultTheme.js

```diff
@@ -1,11 +1,22 @@
 const DefaultTheme = {
   unit: 8,
-  responsive: {},
+
+  responsive: {
+    mediumAndAbove: '@media (min-width: 744px)',
+    largeAndAbove: '@media (min-width: 1128px)',
+  },
+
+  constants: {
+    DEFAULT_HANDLE_WIDTH: 1.5,
+    BACKGROUND_HEIGHT: 0.25,
+  },
+
   color: {
     black: 'black',
     white: '#fcfcfc',
     grey: '#d8d8d8',
-    teal: '#abc4e8',
+    focus: '#abc4e8',
```

Comment:
I think what I meant instead is that we would have:
```
 teal: '#abc4e8',
```
and then also have:
```
focus: this.teal,
progressBar: this.teal,
```
or whatever. 

Alternatively, you could have:
```
const core = {
  black: 'black',
  white: '#fcfcfc',
  grey: '#d8d8d8',
  teal: '#abc4e8',
  lightgrey: '#dbdbdb',
};

const DefaultTheme = {
  ...
  color: {
    ...core,
    focus: core.teal,
    textDisabled: core.lightgray,
    ...
  },
};
```

but maybe that's overengineering it and not super helpful.